packages/data/src/components/use-select/index.js

```diff
@@ -188,20 +190,16 @@ export default function useSelect( mapSelect, deps ) {
 			return;
 		}
 
+		if ( selectorRan ) {
+			latestMapOutput.current = mapOutput;
+		}
```

Comment:
It's probably just for aesthetics sake, but I'd rather keep the assignments to `latestMapOutput` and `latestMapOutputError` close to each other.